lang/it.json

```diff
@@ -623,8 +624,8 @@
     "badgeGallery": {
       "label": "Vetrina di {PLAYER}",
       "loading": "Caricamento...",
-      "bp": "{BP} PM",
-      "count": "{COUNT} Medaglie"
+      "bp": "{BP} BP",
+      "count": "{COUNT} Badges"

```

Comment:
Are these supposed to be translated?

Reply:
I think so, "Badges" should be translated to "Medaglie"

PS: Sorry for the 1 year late reply, it's been a while since I opened up GitHub lmao.